cmd/tink-worker/client/client.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
```

Comment:
I'd recommend swapping this library for the stdlib `"errors"`. it provides all the same functionality without a dependency.

Reply:
Thanks, fixed.